updates/12.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS `neg`.`selektion_konvent` (
+  `ID` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `Bezeichnung` VARCHAR(50) CHARACTER SET 'utf8mb4' COLLATE 'utf8mb4_unicode_ci' NOT NULL,
+  PRIMARY KEY (`ID`),
+  UNIQUE (`Bezeichnung`)
+  )
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8mb4
+COLLATE = utf8mb4_unicode_ci;
+
+/*Insert a default value*/
+INSERT INTO `neg`.`selektion_konvent` (Bezeichnung) VALUES ('-');
```

Comment:
I see two options here:
- Either we avoid using an explicit default value and set the references from other tables to NULL in that case (would be the best DB design). However this might cause problems with certain save forms etc.
- Or, instead of INT UNSIGNED we just use INT and explicitly set the ID of the default value to -1, which is at least similar to most other selektion_- tables.

Reply:
I would try to prevent using negative values in the primary key columns.
I have adjusted the dosave.jsp file to handle NULL values from the formular,
so we have a clean db design for this

Reply:
As we discussed i will change the column definition from INT UNSIGNED to INT and will insert a default value with the id -1.